chart/templates/web.yaml

```diff
@@ -56,12 +56,14 @@ spec:
           containerPort: 8084
         - name: admin-http
           containerPort: 9994
+        volumeMounts:
+        - name: config
+          mountPath: /var/run/linkerd/config
```

Comment:
Looks like one additional test fixture has been added on master since you submitted this change, and that fixture needs to be updated to pass integration tests. Can you merge the latest master into your branch, and then run:

```
go test ./cli/cmd -update
```

That should result in an update to `cli/cmd/testdata/upgrade_default.golden`.